eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -135,6 +135,22 @@ export class UnwrappedQuestionnaireDesignPage extends Component {
     );
   };
 
+  canAddQuestionPage() {
+    const {
+      match: {
+        params: { entityName },
+      },
+      loading,
+    } = this.props;
+
+    return !loading && entityName !== "introduction";
+  }
+
+  canAddCalculatedSummaryPage() {
+    // Criteria for adding summary page same as question page at the moment
```

Comment:
I don't think the comment is necessary. Maybe we could come up with a better method name? So its obvious they are the same logic

Reply:
I've amalgamated the 2 methods into 1